src/main/java/org/ilgcc/app/submission/actions/SaveCCAPStartDate.java

```diff
@@ -29,7 +29,10 @@ public void run(Submission submission, String id) {
         if (earliestCCAPStart.isBlank()) {
```

Comment:
I think we want to set the formatted date when the earliestCCAPStart is null or blank

```suggestion
         if (earliestCCAPStart == null || earliestCCAPStart.isBlank()) {
```